infra/dockerfile.commit_retrieval

```diff
@@ -4,18 +4,18 @@ COPY infra/git_hyper_blame_patch /root/git_hyper_blame_patch
 
 # Mercurial need Python2 :(
 # git and git-hyper-blame (from depot_tools) are required by the annotate pipeline.
-RUN apt-get update && \
-    apt-get install -y python python-pip git curl && \
+RUN apk update && apk upgrade && \
+    apk add --no-cache gcc g++ libgomp lapack-dev git libexecinfo-dev make libc6-compat && \
+    apk add python-dev py-pip curl && \
     python2 -m pip install --disable-pip-version-check --no-cache-dir mercurial==4.8 && \
     hg clone -r 0d68b0deeca2 https://hg.mozilla.org/hgcustom/version-control-tools /version-control-tools/ && \
-    rm -r /version-control-tools/.hg /version-control-tools/ansible /version-control-tools/docs /version-control-tools/testing && \
+    rm -rf /version-control-tools/.hg /version-control-tools/ansible /version-control-tools/docs /version-control-tools/testing && \
```

Comment:
Why this change? We want to know if one of the directories doesn't exist, so we can stop removing it

Reply:
This is a mistake. Rectified in new PR. Thanks for pointing out.